exporter/loadbalancingexporter/resolver_dns.go

```diff
@@ -104,6 +107,8 @@ func (r *dnsResolver) periodicallyResolve() {
 			ctx, cancel := context.WithTimeout(context.Background(), r.resTimeout)
 			if _, err := r.resolve(ctx); err != nil {
 				r.logger.Warn("failed to resolve", zap.Error(err))
+			} else {
+				r.logger.Debug("resolved successfully")
```

Comment:
holdover from debugging?

Reply:
I thought it might be good to leave in to confirm things are working at the debug level. But I can certainly take it out.

Reply:
So is it convention to not have debug logs? Or are you just concerned there's not much information in the log?

Reply:
I'm fine with leaving it